src/components/DateRangePicker/SelectionPane.js

```diff
@@ -0,0 +1,128 @@
+const React = require('react');
+const moment = require('moment');
+const PropTypes = require('prop-types');
+
+const DefaultRanges = require('../DateRangePicker/DefaultRanges');
+
+const StyleConstants = require('../../constants/Style');
+
+class SelectionPane extends React.Component {
+  static propTypes = {
+    currentDate: PropTypes.string,
+    defaultRanges: PropTypes.array,
+    handleDateBoxClick: PropTypes.func,
+    isLargeOrMediumWindowSize: PropTypes.bool,
+    primaryColor: PropTypes.string,
+    selectedBox: PropTypes.string,
+    selectedEndDate: PropTypes.number,
+    selectedStartDate: PropTypes.number,
+    setCurrentDate: PropTypes.func
+  };
+
+  state = {
+    selectedBox: 'from'
+  }
+
+  _handleDateBoxClick = (date, selectedBox) => {
+    this.props.handleDateBoxClick(date, selectedBox);
+  }
+
+  render () {
+    const styles = this.styles();
+    const { selectedStartDate, selectedEndDate, isLargeOrMediumWindowSize } = this.props;
+
+    return (
+      <div style={styles.container}>
+        <div style={{ borderBottom: isLargeOrMediumWindowSize ? 'none' : '1px solid ' + StyleConstants.Colors.FOG }}>
+          <label style={styles.boxLabel}>From</label>
+          <div onClick={() => this._handleDateBoxClick(selectedStartDate, 'from')} style={Object.assign({}, styles.dateSelectBox, this.props.selectedBox === 'from' ? styles.selectedDateSelectBox : null)}>{selectedStartDate ? moment.unix(selectedStartDate).format('MMM D, YYYY') : 'Select Start Date'}</div>
+
+          <label style={styles.boxLabel}>To</label>
+          <div onClick={() => this._handleDateBoxClick(selectedEndDate, 'to')} style={Object.assign({}, styles.dateSelectBox, this.props.selectedBox === 'to' ? styles.selectedDateSelectBox : null)}>{selectedEndDate ? moment.unix(selectedEndDate).format('MMM D, YYYY') : 'Select End Date'}</div>
+        </div>
+        <div>
+          <div style={Object.assign({}, styles.defaultRangesTitle, { color: this.props.primaryColor })}>
+            Select a Range
+          </div>
+          <DefaultRanges {...this.props} styles={styles} />
+        </div>
+      </div>
+    );
+  }
+
+  styles = () => {
+    const { isLargeOrMediumWindowSize } = this.props;
```

Comment:
I think including the isLargeOrMediumWindowSize here directly instead of passing it in via props will make this component more portable.